### PR TITLE
Document Hip in the README, not urllib3

### PR DESCRIPTION
**Pull request by pquentin**
This is a first draft as requested by @njsmith in https://gitter.im/python-trio/hip?at=5e268f6269a7b51d4d54ecae.

You can look at the output here: https://github.com/pquentin/hip/tree/hip-readme

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,110 +1,92 @@
...
     :target: https://github.com/psf/black
 
-urllib3 is a powerful, *sanity-friendly* HTTP client for Python. Much of the
-Python ecosystem already uses urllib3 and you should too.
-urllib3 brings many critical features that are missing from the Python
-standard libraries:
+hip is a new Python HTTP client for everybody. It supports synchronous Python (just like requests does), but also Trio, asyncio and Curio.
 
-*Currently not supporting SOCKS proxies, pyOpenSSL and SecureTransport.*
+hip is robust as it is based on urllib3 and uses its extensive test suite that was refined over the years. I also shares most urllib3 features:
```
Comment (medariox) [inline README.rst]:
```suggestion
hip is robust as it is based on urllib3 and uses its extensive test suite that was refined over the years. It also shares most urllib3 features:
```

Comment (pquentin) [reply README.rst]: Thanks!

File: README.rst [hunk 2]
```diff
@@ -1,110 +1,92 @@
...
 
-.. image:: https://readthedocs.org/projects/urllib3/badge/?version=latest
-        :alt: Documentation Status
-        :target: https://urllib3.readthedocs.io/en/latest/
-
-.. image:: https://img.shields.io/codecov/c/github/urllib3/urllib3.svg
+.. image:: https://img.shields.io/codecov/c/github/python-trio/hip.svg
         :alt: Coverage Status
-        :target: https://codecov.io/gh/urllib3/urllib3
+        :target: https://codecov.io/gh/python-trio/hip
```
Comment (RatanShreshtha) [inline README.rst]: Here the img shield badge is showing 100% coverage but codecov is showing 99.6%, maybe we should use the badge by codecov

File: README.rst [hunk 3]
```diff
@@ -1,110 +1,92 @@
...
+        :target: https://gitter.im/python-trio/hip
 
 .. image:: https://img.shields.io/badge/code%20style-black-000000.svg
     :target: https://github.com/psf/black
 
-urllib3 is a powerful, *sanity-friendly* HTTP client for Python. Much of the
-Python ecosystem already uses urllib3 and you should too.
-urllib3 brings many critical features that are missing from the Python
-standard libraries:
+hip is a new Python HTTP client for everybody. It supports synchronous Python (just like requests does), but also Trio, asyncio and Curio.
```
Comment (sethmlarson) [inline README.rst]: Let's use "Hip" instead of "hip" for the name of the library in copy?

File: README.rst (hunk 2 above)
Comment (pquentin) [reply README.rst]: Oh that's bad, but do you know where we can find a badge with the correct information?

Comment (njsmith) [reply README.rst]: For Trio we're using this URL: https://codecov.io/gh/python-trio/trio/branch/master/graph/badge.svg

So maybe: https://codecov.io/gh/python-trio/hip/branch/master/graph/badge.svg
Which looks like: coverage

Comment (pquentin) [reply README.rst]: Thanks! Fixed.

File: README.rst (hunk 3 above)
Comment (pquentin) [reply README.rst]: Oops, sorry for Hip, fixed. I tried adding a disclaimer but I don't know how to make it look good in a .rst file. Maybe it should just be simple text.

File: README.rst (hunk 1 above)
Comment (pquentin) [reply README.rst]: @medariox Also, welcome to the Hip project! Would you be interested in helping us out?

**Comment by pquentin** [comment]
I've taken into account all comments, would be happy to continue to fix things. We can also merge and continue to tweak the README, it's just intended to be a starting point.

**Pull request merged**